charts/contact-location-by-un-region.js

```diff
@@ -47,71 +47,57 @@ jQuery(document).ready(function () {
 
     let polygonSeries = chart.series.push(
       am5map.MapPolygonSeries.new(root, {
-        geoJSON: am5geodata_unRegionsLow
+        geoJSON: am5geodata_unRegionsLow,
+        valueField: "value",
+        calculateAggregates: true
       })
     );
 
+    polygonSeries.set('heatRules', [{
+      key: "fill",
+      target: polygonSeries.mapPolygons.template,
+      min: am5.color('#9BC8FE'),
+      max: am5.color('#25529A'),
```

Comment:
@kodinkat here is using amcharts inbuilt min max color gradient system.

Reply:
@corsacca Ah, just seeing this and sanity checked locally - Holds true for me also... Thanks for that!